stage.sh

```diff
@@ -86,7 +86,7 @@ done
 
 # Sync product images
 echo " + Copying product images"
-aws s3 sync ./images s3://${BUCKET}/${S3PATH}images --only-show-errors $S3PUBLIC
+aws s3 sync s3://rds-dataset-releases/1.3/images/  s3://${BUCKET}/${S3PATH}images/ $S3PUBLIC
```

Comment:
This would need to be updated before merging in.
@james-jory  please let us know a bucket where you would like us to copy all of the image data

Reply:
I created the bucket `retail-demo-store-assets` and sync'd the images from `rds-dataset-releases` to the `/datasets/1.3/images/` folder.

So the following line in `stage.sh` should work.

```bash
aws s3 sync s3://retail-demo-store-assets/datasets/1.3/images/  s3://${BUCKET}/${S3PATH}images/ $S3PUBLIC
```

Reply:
Changed bucket to an existing one (`retail-demo-store-code`) that is already on the allow list.